### PR TITLE
Make Inspector Live-View Sticky

### PR DESCRIPTION
**Pull request by strseb**
## Description

https://github.com/mozilla-mobile/mozilla-vpn-client/assets/9611612/8f76f0e9-620a-4578-b110-57d8b319d5d3

### CODE REVIEW

**Review by MattLichtenstein — APPROVED** [review]
image

File: tools/inspector/views/view-ui.js [hunk 1]
```diff
@@ -85,6 +85,10 @@ export class ViewUi extends LitElement {
         position: sticky;
         top: 20px;
     }
+    live-view{
+      position:sticky;
```
Comment (lesleyjanenorton) [inline tools/inspector/views/view-ui.js:89]:
```suggestion
      position: sticky;
```

File: tools/inspector/views/view-ui.js [hunk 2]
```diff
@@ -85,6 +85,10 @@ export class ViewUi extends LitElement {
...
+      top:0px;
```
Comment (lesleyjanenorton) [inline tools/inspector/views/view-ui.js:90]:
```suggestion
      top: 0px;
```

**Review by lesleyjanenorton — APPROVED** [review]
🤩

**Pull request merged**